server/cmd/mmctl/commands/constants/urls.go

```diff
@@ -0,0 +1,5 @@
+package constants
```

Comment:
`constants` is not a good package names (https://go.dev/blog/package-names#bad-package-names). Let's just move the `const` into the `commands` package.